playbook.yml

```diff
@@ -3,79 +3,87 @@
 - name: Install thirdparties
   hosts: trento-server
   become: true
+  pre_tasks:
+    - name: Check SLES distribution and version
+      ansible.builtin.fail:
+        msg: "This playbook only runs on SLES for SAP 15 SP3 or above. Detected: {{ ansible_distribution }} {{ ansible_distribution_version }}"
+      when: >
+        (ansible_distribution != "SLES_SAP") or
+        (ansible_distribution_version is version('15.3', '<'))
+
   tasks:
-   - name: Install installation prerequisites
-     community.general.zypper:
-      name:
-       - gcc
-       - python3-devel
-       - sudo
+    - name: Install installation prerequisites
```

Comment:
I think we need some yaml linter here

Reply:
I would go for the double-space indenting that is used more extensively. Should we enforce this in the CI?

Reply:
I don't really care, as long as we don't do this "out of the blue" changes